docs/io/fits/index.rst

```diff
@@ -589,6 +589,42 @@ lines to accomplish the same behavior::
 This will write a single HDU to a FITS file without having to manually
 encapsulate it in an :class:`HDUList` object first.
 
+Creating a Multi-Extension Image File
+"""""""""""""""""""""""""""""""""""""
+
+In the previous example we created an :class:`HDUList` object with a single
+extension (a :class:`PrimaryHDU`). To create a file with multiple extensions we need to
+use :class:`ImageHDU`\s and append them to an :class:`HDUList`.
+
+First, let's create some more data::
+
+    >>> n2 = np.random.random((3, 3)) # A random 3x3 array
```

Comment:
Since the actual data isn't important, I would prefer to use `np.zeros`, `np.ones`, `np.arange`, or `np.linspace`.

Generating lots of random numbers can be slow and unnecessarily slows down examples.